src/handler.rs

```diff
@@ -261,7 +261,15 @@ impl EventHandler for Handler {
     /// window, then Galileo will respond instructing the user to wait longer.
     async fn message(&self, ctx: Context, message: Message) {
         // Check whether we should proceed.
-        let message_info = MessageInfo::new(&message, &ctx).unwrap();
+        let message_info = match MessageInfo::new(&message, &ctx) {
```

Comment:
this looks great 👍

eventually it could be cool to expose a metric counting messages observed / parsed, but that's another item for the rainy day bucket 🙂